test/suites/integration/nodeclass_test.go

```diff
@@ -87,10 +85,10 @@ var _ = Describe("NodeClass IAM Permissions", func() {
 
 			env.ExpectCreated(nodeClass)
 			Eventually(func(g Gomega) {
```

Comment:
What's the default timeout here?

Reply:
15m -- it's set by SetEventuallyTimeout

Reply:
We could do 10m but I'm inclined to not try to be cute here and just stick with the default

Reply:
The issue here is that IAM permission dropping (due to service caching) can take up 1o 5m (that's what I've seen in testing) to fully re-resolve